packages/celotool/src/lib/blockscout.ts

```diff
@@ -165,7 +71,7 @@ export async function createDefaultIngressIfNotExists(celoEnv: string, ingressNa
     console.info(`Creating ingress ${celoEnv}-blockscout-web-ingress`)
     const ingressFilePath = `/tmp/${celoEnv}-blockscout-web-ingress.yaml`
     const ingressResource = `
```

Comment:
Is there no way to remove the ingress resource js template and use a regular helm template?